cargo-dist/tests/integration-tests.rs

```diff
@@ -194,6 +194,37 @@ path-guid = "BFD25009-65A4-4D1E-97F1-0030465D90D6"
     })
 }
 
+#[test]
+fn axolotlsay_user_publish_job() -> Result<(), miette::Report> {
+    let test_name = _function_name!();
+    AXOLOTLSAY.run_test(|ctx| {
+        let dist_version = ctx.tools.cargo_dist.version().unwrap();
+        ctx.patch_cargo_toml(format!(r#"
+[workspace.metadata.dist]
+cargo-dist-version = "{dist_version}"
+installers = ["shell", "powershell", "homebrew", "npm"]
+tap = "axodotdev/homebrew-packages"
+publish-jobs = ["homebrew", "./custom-task"]
```

Comment:
how hard would it be to make this n=2 to make sure the loop expands pretty and there's no weird conflicts?

Reply:
Good call, made it two custom tasks.